bd-crash-handler/src/lib.rs

```diff
@@ -265,6 +267,20 @@ impl Monitor {
             continue;
           },
         };
+
+        let timestamp = path
+          .file_name()
+          .and_then(|name| name.to_str())
+          .and_then(|name| {
+            name.split('_').next().and_then(|timestamp| {
+              let Ok(timestamp) = timestamp.parse() else {
+                log::debug!("Failed to parse timestamp from file name: {:?}", name);
+                return None;
```

Comment:
should we return Now as a fallback?

Reply:
I do already elsewhere but I can move the handling here since I didn't end up having to test it at this level